app/src/main/java/com/google/samples/apps/sunflower/HomeViewPagerFragment.kt

```diff
@@ -61,4 +61,13 @@ class ViewPagerFragment : Fragment() {
 
         return binding.root
     }
+
+    /** Return the list of tab titles to pass to SunflowerPagerAdapter **/
+    private fun getTabTitles(): List<String> {
+        val tabTitles: MutableList<String> = arrayListOf()
```

Comment:
What do you think about using a HashMap?  That way the case statement in `getPageTitle` can be simplified to `tabTitles.get(position)`?

Reply:
I agree, it would make for faster indexing too

Reply:
updated